Cubical/HITs/SetTruncation/Properties.agda

```diff
@@ -75,6 +76,127 @@ elim3 Bset g = elim2 (λ _ _ → isSetΠ (λ _ → Bset _ _ _))
                      (λ a b → elim (λ _ → Bset _ _ _) (g a b))
 
 
+-- the recursor for maps into groupoids following the "HIT proof" in:
```

Comment:
Isn't this already merged?

Reply:
Yeah sorry, I just kept working on the same branch and didn't merge with the updated master branch. Should be fine now.